core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -773,6 +774,64 @@ class DAGSchedulerSuite
     assertDataStructuresEmpty()
   }
 
+  ignore("no concurrent retries for stage attempts (SPARK-7308)") {
```

Comment:
ignored b/c it spawns 10 executors, takes about 2 mins on my laptop, and makes everything pretty sluggish -- I didn't want to swamp jenkins.  I tried a variety of permutations and this consistently demonstrated the problem for me, but maybe we can pare this down some.  (Or maybe we need another home for tests like this?)